src/Microsoft.OData.Client/Serialization/PrimitiveXmlConverter.cs

```diff
@@ -858,10 +858,36 @@ internal override string ToString(object instance)
         }
     }
 
+    /// <summary>
+    /// Convert between primitive types Edm.Date and string

```

Comment:
Should this comment be updated to mention DateOnly?

Reply:
I updated the comment to fresh the 'DateOnly' C# type. #Resolved